preflight/preflight.sh

```diff
@@ -148,12 +150,13 @@ echo "Success"
 echo -n "Determining external network CIDR..."
 BARNET=`/usr/bin/ipcalc -n "$(/usr/sbin/ip -o addr show|grep baremetal|grep -v inet6|awk {'print $4'})"|cut -f2 -d=`
 BARCIDR=`/usr/bin/ipcalc -p "$(/usr/sbin/ip -o addr show|grep baremetal|grep -v inet6|awk {'print $4'})"|cut -f2 -d=`
+wc=$((wc-1))
```

Comment:
Having a comment why you are doing this -1 way be helpful.

I assume it is because the last worker will be the deployment host and should not count as a worker for the initial deployment.

If so,